streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueStore.java

```diff
@@ -66,10 +66,10 @@ public String name() {
         return name;
     }
 
-    @Deprecated
     @Override
-    public void init(final ProcessorContext context,
+    public void init(final StateStoreContext stateStoreContext,
                      final StateStore root) {
+        final ProcessorContext context = StoreToProcessorContextAdapter.adapt(stateStoreContext);
```

Comment:
Do we need `context` at all? Seem we can use `stateStoreContext` below got call `appConfigs()` and `register()` ?